openmsupply/open-msupply
# 1362 disable the stock charges tax entry when the total stock charges are $0

Fixes #1362

# 👩🏻‍💻 What does this PR do? 
Disable service tax edit for inbound and outbound shipments if the total is 0

# 🧪 How has/should this change been tested? 
- Add empty service tax line - see that tax pen is still disabled.
- Add service line with total - see that tax pen has been enabled 

Do we want to do this with stock charges as well? Issue is only talking about service charge...
